Submit to nature address reviewers (#56)
Nikoleta-v3/Memory-size-in-the-prisoners-dilemma: main.tex

\newtheorem{lemma}[theorem]{Lemma}
\def\arraystretch{1.5}

\title{Best response memory-one strategies with a theory of mind: the limitations

> of extortion and restricted memory.}

Reviewer: ```suggestion
\title{A theory of mind: Best responses to memory-one strategies. The limitations
of extortion and restricted memory.}
```

with and without self interactions are given in
Figure~\ref{fig:sse_distributions}. Moreover, a statistics summary of the SSE
distributions is given in Table~\ref{table:sserror_stats}.

\begin{table}[!htbp]
    \begin{center}
    \resizebox{.8\columnwidth}{!}{%
    \begin{tabular}{lrrrrrrrrrrr}
        \toprule
        & mean & std  & 5\% & 50\% &  95\% & max & median & skew & kurt\\
        \midrule
    \textbf{Tournament without self interactions} & 0.34  & 0.40  & 0.028  & 0.17  &
    1.05  & 2.47  & 0.17  & 1.87 & 3.60 \\
    \textbf{Tournament with self interactions} & 0.17 & 0.23 & 0.01 &
    0.12 & 0.67 & 1.53 & 0.12 & 3.42 & 1.92 \\
        \bottomrule
    \end{tabular}}
    \end{center}
    \caption{SSE of best response memory-one when \(N=2\)}\label{table:sserror_stats}
\end{table}

\begin{figure}[!htbp]
    \begin{subfigure}{0.47\textwidth}
        \begin{center}
            \includegraphics[width=\linewidth]{img/best_respones_sserror.pdf}
        \end{center}
        \caption{SEE distribution for best response in tournaments without self interactions.}
    \end{subfigure}\hfill
    \begin{subfigure}{0.47\textwidth}
        \begin{center}

>             \includegraphics[width=\linewidth]{img/evo_sserror.pdf}

Author: https://github.com/Nikoleta-v3/Memory-size-in-the-prisoners-dilemma/issues/57

        \end{center}
        \caption{SEE distribution for best response in tournaments with self interactions.}
    \end{subfigure}
    \caption{SEE distributions for best response in tournaments without and with self interactions.}\label{fig:sse_distributions}
\end{figure}

For the best response in tournaments that do not include self interactions the
distribution of SSE is skewed to the left, indicating that the best response
does exhibit ZDs behaviour and so could be extortionate, however, the best
response is not uniformly a ZDs. A positive measure of skewness and kurtosis,
and a mean of 0.34 indicate a heavy tail to the right. Therefore, in several
cases the strategy is not trying to extort its opponents. More specifically,
the best response memory-one with a theory of mind is being more adaptable than a ZDs.
In~\cite{Hilbe2018} a similar behaviour is refereed to as the \textit{partners
strategy}. Partners
strategy aims to share the payoff for mutual cooperation, but it is ready

> to fight back when being exploited. In~\cite{Hilbe2018} they showed that partner strategies are

Reviewer: ```suggestion
to fight back when being exploited. In~\cite{Hilbe2018} it is shown that partner strategies are
```

selected in evolutionary settings when populations are large and relationships

> stable and they allow for the evolution of cooperation. A best response memory-one strategy

Reviewer: Let's have a paragraph break here.

in evolutionary settings has not been incorporated in the study of this manuscript, however,

> the best response memory-one strategy with self interactions can used to approximate

Reviewer: ```suggestion
the best response memory-one strategy with self interactions can used to approximate % TODO Vince to work on this a bit
```

Essentially we're saying:

- Best response mem1: adapt they are sometimes ZD and sometimes not;
- There is a paper by Hilbe that described a strategy designed to do the same thing and was evolutionary stable.
- That's cool.

mutual cooperation as well as exploitation which is line with the results of
\cite{Hilbe2018}.

The difference between best responses in tournaments without and with self interactions
is further explored by Fig.~\ref{fig:behaviour_violin_plots}.
Though, no statistically significant differences have been found, from
Fig.~\ref{fig:behaviour_violin_plots}, it seems that the best
response that incorporate self interactions has a higher median $p_2$; which corresponds to the probability of cooperating
after receiving a defection. Thus, they are more likely to forgive after
being tricked. This is due to the fact that they could be playing against
themselves, and they need to be able to forgive so that future cooperation can
occur.

\begin{figure}[!htbp]
    \centering
    \includegraphics[width=.9\textwidth]{img/behaviour_violin_plots.pdf}
    \caption{Distributions of \(p^*\) for best responses in tournaments and
    evolutionary settings. The medians, denoted as \(\bar{p}^*\), for tournaments
    are \(\bar{p}^* = (0, 0, 0, 0)\), and for evolutionary settings
    \(\bar{p}^* = (0, 0.19, 0, 0)\).}
    \label{fig:behaviour_violin_plots}
\end{figure}

> The second limitation this paper argues that memory-one extortionate strategies

Reviewer: This sentence needs fixing.

> \section{Discussion}

Reviewer: Let's include the stability of defection more prominently here.

```
One of the main results of this paper is a closed form for the utility of a strategy (blablabla) this not only allows for the analysis we've discussed but can also be used to prove other theoretic results such as

THEORM

PLOT
```

Reviewer: Put this where you are currently discussing it.
